fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -195,16 +206,104 @@ countries:
                         time: T
                     colormap: *raincm
                     lower_is_worse: yes
+                chirps-precip-djf:
+                    label: CHIRPS Precip
+                    description: CHIRPS average precipitation over the season
+                    path: madagascar/chirps-precip-djf.zarr
+                    var_names:
+                        obs: precipitation
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: *raincm
+                    lower_is_worse: yes
+                ndvi:
+                    label: NDVI
+                    description: MODIS NDVI average over the season
+                    path: madagascar/ndvi-djf.zarr
+                    var_names:
+                        obs: NDVI
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: *raincm
+                    lower_is_worse: yes
+                    format: number4
+                wrsi:
+                    label: WRSI
+                    path: madagascar/wrsi-djf.zarr
+                    var_names:
+                        obs: do
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: *raincm
+                    lower_is_worse: yes
+                spi:
+                    label: ENACTS MON SPI
+                    description: SPI calculated from ENACTS monitoring product
+                    path: madagascar/enacts-mon-spi-djf.zarr
+                    var_names:
+                        obs: spi
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: *raincm
+                    lower_is_worse: yes
+                    format: number3
+                subseas-dry-tercile-snov-l22-djf:
+                    label: November biweekly forecast, 3wk lead
+                    description: Biweekly rainfall forecast issued in early November for the two week period starting a week after the issue date. Probability of the driest tercile.
```

Comment:
"starting two weeks after the issue date". You copied the description from Niger which uses L=15 whereas here we're using L=22.